pkg/kapp/cmd/appgroup/delete.go

```diff
@@ -55,7 +55,7 @@ func (o *DeleteOptions) Run() error {
 		return fmt.Errorf("Expected group name to be non-empty")
 	}
 
-	supportObjs, err := cmdapp.FactoryClients(o.depsFactory, o.AppGroupFlags.NamespaceFlags, cmdapp.ResourceTypesFlags{}, o.logger)
+	supportObjs, err := cmdapp.FactoryClients(o.depsFactory, o.AppGroupFlags.NamespaceFlags.Name, cmdapp.ResourceTypesFlags{}, o.logger)
```

Comment:
Are we saying that `--app-namespace` won't apply to app groups? I am not sure if we should do that. 
Ideally, we should allow setting of a location for metadata cm for app groups as well. Do we have other flags that only apply to apps but not app groups?

Reply:
Added app-namespace for app-group.